test/backend/fused_op.in.cpp

```diff
@@ -581,157 +581,170 @@ NGRAPH_TEST(${BACKEND_NAME}, depth_to_space)
     test_case.run();
 }
 
-NGRAPH_TEST(${BACKEND_NAME}, normalize_across_chw_scalar_scale_4d)
+NGRAPH_TEST(${BACKEND_NAME}, normalize_across_chw_4d)
 {
     Shape data_shape{1, 2, 3, 4};
     auto data = make_shared<op::Parameter>(element::f32, data_shape);
-    auto scale = make_shared<op::Parameter>(element::f32, Shape{});
-    bool across_spatial{false};
-    bool channel_shared{true};
+    const auto axes = make_shared<op::Constant>(element::u64, Shape{3}, vector<int64_t>{1, 2, 3});
     float eps{1e-6f};
+    auto eps_mode = op::NormalizeL2::EpsMode::ADD;
 
-    auto normalize = make_shared<op::Normalize>(data, scale, across_spatial, channel_shared, eps);
-    auto function = make_shared<Function>(NodeVector{normalize}, ParameterVector{data, scale});
+    auto normalize = make_shared<op::NormalizeL2>(data, axes, eps, eps_mode);
+    auto function = make_shared<Function>(NodeVector{normalize}, ParameterVector{data});
 
     auto test_case = test::NgraphTestCase(function, "${BACKEND_NAME}");
 
     vector<float> input_data(shape_size(data_shape));
     iota(begin(input_data), end(input_data), 1);
 
     test_case.add_input<float>(input_data);
-    test_case.add_input<float>({2.f});
 
     test_case.add_expected_output<float>(
-        data_shape, {0.02857143f, 0.05714286f, 0.08571429f, 0.11428571f, 0.14285714f, 0.17142857f,
-                     0.2f,        0.22857143f, 0.25714286f, 0.28571429f, 0.31428571f, 0.34285714f,
-                     0.37142857f, 0.4f,        0.42857143f, 0.45714286f, 0.48571429f, 0.51428571f,
-                     0.54285714f, 0.57142857f, 0.6f,        0.62857143f, 0.65714286f, 0.68571429f});
+        data_shape, {0.01428571f, 0.02857143f, 0.04285714f, 0.05714286f, 0.07142857f, 0.08571429f,
```

Comment:
Where did these magic numbers come from?

Reply:
I implemented python script to calculate expected results

Reply:
Can you add the python script to `test/ref_generators`?

Reply:
Sure ;)  I added script to `test/ref_generators/generate_normalize_l2_ref.py`